lib/pages/view-post-page.js

```diff
@@ -68,7 +68,7 @@ export default class ViewPostPage extends BaseContainer {
 
 	commentEventuallyShown( comment ) {
 		const self = this;
-		const commentSelector = By.xpath( `//p[text() = "${comment}"]` );
+		const commentSelector = By.css( `p:contains("${comment}")` );
```

Comment:
Turns out it is! I just researched into it a little bit and found out it had been deprecated. I'll look at another solution here instead. Thanks!